src/ui/components/lineage/index.css

```diff
@@ -461,11 +455,13 @@
   font-family: 'Rubik';
   font-style: normal;
   font-weight: 400;
-  font-size: 18px;
+  font-size: 12px !important;
```

Comment:
<img width="826" alt="grafik" src="https://github.com/zenml-io/zenml-dashboard/assets/43843195/b492f80f-87f6-4043-aa13-8fecfc97bb8a">
This change makes the font on all other tabs smaller as well. Is this intended?

Reply:
yes. it will make the design consistent for all tabs which use the lineage component